style.css

```diff
@@ -461,6 +461,123 @@ span {
   margin-right: 35px;
 }
 
+.modal-container {
+  display: none;
+  flex-direction: column;
+  align-content: center;
+  width: 100%;
+  padding: 10px;
+  background-color: #979493;
+  overflow: hidden;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.modal {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  margin-left: 16px;
+  max-width: 90%;
+  height: 100vh;
+  background-color: #fff;
+}
+
+.modal-pop {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.modal-pop img {
+  width: 12px;
+  height: 12px;
+  margin-right: 32px;
+}
+
+.modal-container h3 {
+  font-family: "Roboto", sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  margin-top: 16px;
+  margin-bottom: 20px;
+}
+
+.modal p {
+  color: #344563;
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  line-height: 24px;
+  font-size: 16px;
+  align-items: center;
+  margin-top: 24px;
+  margin-bottom: 16px;
+  width: 95%;
+}
+
+.modal-btn {
+  display: flex;
+  width: 95%;
+  justify-content: space-between;
+}
+
+.modal-btn img {
+  margin-left: 8px;
+}
+
+.btn-1,
+.btn-2 {
+  height: 42px;
+  background-color: #ff6b00;
+  text-align: center;
+  margin-left: 2px;
+  margin-top: 8px;
+  border: none;
+  padding: 13px;
+  color: white;
+  font-size: 17px;
+  line-height: 24px;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+}
```

Comment:
- [ ] Kindly make sure your button contents are centered in order to meet the provided Figma design :wink:

    - **YOUR DESIGN**
     ![image](https://user-images.githubusercontent.com/87197412/210881744-bc8ff279-1f63-4fe8-9ec9-61c77daed802.png)

    - **FIGMA DESIGN**
   ![image](https://user-images.githubusercontent.com/87197412/210881789-3740b9ba-93ff-4f7a-bc41-0be03af59765.png)